src/plugins/data/common/search/es_search/shim_abort_signal.ts

```diff
@@ -17,7 +17,13 @@
  * under the License.
  */
 
-import { TransportRequestPromise } from '@elastic/elasticsearch/lib/Transport';
+/**
+ * @internal
+ * TransportRequestPromise extends base Promise with an "abort" method
+ */
+export interface TransportRequestPromise<T> extends Promise<T> {
```

Comment:
Why not import this as it was before?

Reply:
Currently we use shim_abort_signal for queries from the client side too. These queries works out of the elasticsearch lib and not sure that we should import types from the previous place.